### DISCUSSION TITLE
Remove header method?

### DISCUSSION THREAD

**Discussion by dalisoft** [uNetworking/uWebSockets.js#150] [Q&A] [answered]
Hi everyone. I'm building own Express/Fastify-like library built on top of uWebSockets.js.

But now i'm need `res.removeHeader` or similar function. Is there any function?

Thanks

**Comment by ghost** [comment, marked as answer]
You cannot "unwrite" a written header.

**Comment by dalisoft** [comment]
Any way to set them to empty as least? For remove cookie this method need

**Comment by ghost** [comment]
You **cannot** "unwrite" a written header.

**Comment by dalisoft** [comment]
Okay. Thanks
